packages/core/integration-tests/test/scope-hoisting.js

```diff
@@ -251,7 +251,7 @@ describe('scope hoisting', function() {
       assert.equal(output, 8);
     });
 
-    it('supports dynamic import syntax for code splitting', async function() {
+    it.skip('supports dynamic import syntax for code splitting', async function() {
```

Comment:
These are broken. I don't have much context on scopehoisting and was hoping @devongovett and @mischnic could help out.

Reply:
I can dig in tomorrow probably